src/track/track.cpp

```diff
@@ -215,19 +218,16 @@ void Track::importMetadata(
     //
     // TODO: Import Serato metadata within the locking scope and not
     // as a post-processing step.
-    auto pBeatsImporter = importedSeratoTags.importBeats();
-    if (pBeatsImporter) {
+    if (pSeratoBeatsImporter) {
         kLogger.debug() << "Importing Serato beats";
-        tryImportBeats(std::move(pBeatsImporter), importedSeratoTags.isBpmLocked());
+        tryImportBeats(std::move(pSeratoBeatsImporter), seratoBpmLocked);
     }
-    auto pCuesImporter = importedSeratoTags.importCueInfos();
-    if (pCuesImporter) {
+    if (pSeratoCuesImporter) {
         kLogger.debug() << "Importing Serato cues";
-        importCueInfos(std::move(pCuesImporter));
+        importCueInfos(std::move(pSeratoCuesImporter));
     }
-    const mixxx::RgbColor::optional_t optColor = importedSeratoTags.getTrackColor();
-    if (optColor) {
-        setColor(*optColor);
+    if (optSeratoColor) {
```

Comment:
What if I uncolor the track color in Serato? Will this unset the track color in Mixxx as well?

https://github.com/mixxxdj/mixxx/pull/3723/commits/8757e066fdf18d3d4d11b180cde15b8e70d297ea#r597616445

(For some reason I can't see this line in the PR diff, only when click on this commit).

Reply:
Ah ok, I guess you already fixed that in your latest commit.